app/Http/Controllers/API/v1/TransportController.php

```diff
@@ -130,56 +163,58 @@ class TransportController extends Controller
      *         )
      *        )
      *     ),
-     * @OA\Response(
-     *         response=404,
-     *         description="Station not found",
-     *     ),
-     * @OA\Response(
-     *         response=502,
-     *         description="Error with our data provider",
-     *     ),
-     * @OA\Response(
-     *         response=422,
-     *         description="Invalid input",
-     *     ),
-     * @OA\Response(response=401, description="Unauthorized"),
-     *     security={
-     *        {"passport": {"create-statuses"}}, {"token": {}}
-     *
-     *     }
+     *      @OA\Response(response=401, description="Unauthorized"),
+     *      @OA\Response(response=404, description="Station not found"),
+     *      @OA\Response(response=422, description="Invalid input"),
+     *      @OA\Response(response=502, description="Error with our data provider"),
+     *      security={{"passport": {"create-statuses"}}, {"token": {}}}
      * )
      *
      * @param Request $request
-     * @param string  $name
+     * @param int     $stationId
      *
      * @return JsonResponse
-     * @see All slashes (as well as encoded to %2F) in $name need to be replaced, preferrably by a space (%20)
      */
-    public function departures(Request $request, string $name): JsonResponse {
+    public function getDepartures(Request $request, int $stationId): JsonResponse {
         $validated = $request->validate([
                                             'when'       => ['nullable', 'date'],
                                             'travelType' => ['nullable', new Enum(TravelType::class)],
                                         ]);
 
+        $timestamp = isset($validated['when']) ? Carbon::parse($validated['when']) : now();
+        $station   = Station::findOrFail($stationId);
+
         try {
-            $trainStationboardResponse = TransportBackend::getDepartures(
```

Comment:
Maybe also mark `TransportBackend::getDepartures` as deprecated + remove its usage in the admin panel + add removal TODO?

Reply:
I'll mark it as deprecated but prefer to remove the usage elsewhere in other PRs.